bots/tests-scan

```diff
@@ -221,7 +221,7 @@ def external_tasks (update, policy):
             statuses = api.statuses(revision)
             login = pull["head"]["user"]["login"]
 
-            have = len(statuses.keys()) > 0
+            have = len(list(statuses.keys())) > 0
```

Comment:
Just `len(statuses)` is enough (no need for `.keys()`), but for sure the `list()` is not necessary.

Reply:
Removed the list. We can do other fixups later.